x/plans/keeper/plan_test.go

```diff
@@ -0,0 +1,280 @@
+package keeper_test
```

Comment:
Need to add tests for:
- `PutPlan()` and the refcounts (i.e. that a plan no longer in use should no "get-able")
- deletion of old versions (and in the right order)

Reply:
done